src/Mod/TechDraw/Gui/ViewProviderDimension.cpp

```diff
@@ -86,6 +86,8 @@ ViewProviderDimension::ViewProviderDimension()
                       "Adjusts the gap between dimension point and extension line");
     ADD_PROPERTY_TYPE(GapFactorASME, (Preferences::GapASME()), group, App::Prop_None,
                       "Adjusts the gap between dimension point and extension line");
+    ADD_PROPERTY_TYPE(LineSpacingISO, (-1.0), group, App::Prop_None,

```

Comment:
Why do we need to define the property here too, if it was already defined in the ui file???

Reply:
The Properties in the ViewProvider are persistent,  they are saved with the document.  Properties in the UI apply to the widget.